config/sources/families/rockchip.conf

```diff
@@ -21,10 +21,12 @@ if [[ "$BOOT_SOC" == "rk3288" ]]; then
 	BOOTSCRIPT="boot-rockchip.cmd:boot.cmd"
 	BOOTENV_FILE='rockchip.txt'
 	OVERLAY_PREFIX='rockchip'
-	UBOOT_TARGET_MAP=";;$SRC/packages/blobs/rockchip/rk3288_boot.bin u-boot-rockchip-with-spl.bin"
+	UBOOT_TARGET_MAP=";;u-boot-rockchip-with-spl.bin"
 
-	BOOTBRANCH='tag:v2022.04'
-	BOOTPATCHDIR='v2022.04'
+	[[ "$BOARD" == "xt-q8l-v10" ]] && UBOOT_TARGET_MAP="ROCKCHIP_TPL=$SRC/cache/sources/rkbin-tools/rk32/rk3288_ddr_400MHz_v1.08.bin;;u-boot-rockchip-with-spl.bin"
```

Comment:
Ok, that's good advice. Actually in first instance I put `UBOOT_TARGET_MAP` in the board conf, but the env variable didn't get overwritten, so I give up fast because had to deal with more urgent rework issues.